pom.xml

```diff
@@ -48,35 +47,53 @@
         </license>
     </licenses>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-all</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-core</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-api</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-            <dependency>
-                <!-- this gives us flexible and easy-to-use logging; just edit logback-custom.xml! -->
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-logback-xml</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-        </dependencies>
-    </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-all</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-core</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-api</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.cloudfoundry</groupId>
+            <artifactId>cloudfoundry-client-lib</artifactId>
+            <version>${cloudfoundry.client.version}</version>
+        </dependency>
+
+        <!-- http://mvnrepository.com/artifact/org.mockito/mockito-all -->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>${mockito.version}</version>
+        </dependency>
+
+        <dependency>
+            <!-- this gives us flexible and easy-to-use logging; just edit logback-custom.xml! -->
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-logback-xml</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-core</artifactId>
+            <version>${brooklyn.version}</version>
+            <classifier>tests</classifier>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
 
     <build>
```

Comment:
@andreaturli thanks for your reply. 
I have deleted the `rat` plugin configuration and I have deleted the licenses of some classes but I did not found any the typical exception ` Too many files with unapproved license`  during the project building (`mvn clean install`). I found the `brooklyn-downstream-parent` pom file (v0.10.0), and I found the `rat` plugin configuration inside `pluginManagement` element . I saw that `brooklyn-downstream-parent` uses `brooklyn-server` pom file as parent. However, I did not find any configuration for the `rat` plugin inside this pom. So, it looks that rat plugin is not being inherit. 
But, I probably lost some detail about the `rat` configuration and my thoughts are wrong.